### PR TITLE
1784-useNewSigelMapping

### LINKED ISSUE #1784: Clean up ISIL-Sigel mapping
See this query to find out problematic ISILs: https://lobid.org/resources/search?q=hasItem.heldBy.label.raw:%22lobid%20Organisation%22

### PR DESCRIPTION
**Pull request by TobiasNx**
This PR introduces a new way for mapping sigel to isil as well as lobid-organisation-Id to label with a mapping file created like this:

http://lobid.org/organisations/search?q=_exists_%3Aisil+OR+type%3ACollection&format=csv:id,isil,sigel,name&size=25000 

And transformed the file to a tsv.

With this file we kick out all sigil/isil's that are not registerd with the https://sigel.staatsbibliothek-berlin.de/

Like "DE-386-32"

Resolves #1784

### CODE REVIEW

**Review by blackwinter — APPROVED** [review]
LGTM. Except for the failing tests...

File: src/test/resources/alma-fix/990050000600206441.json [hunk 1]
```diff
@@ -266,11 +266,11 @@
     "type" : [ "Item", "PhysikalischerTitel" ],
     "currentLocation" : "B4002 / UNASSIGNED",
     "heldBy" : {
-      "id" : "http://lobid.org/organisations/DE-294-22#!",
-      "isil" : "DE-294-22",
-      "label" : "Ruhr-Universität Bochum, Verbundbibliothek IB Fachbibliothek für Geographie, Geologie, Mathematik, Psychologie"
```
Comment (TobiasNx) [inline src/test/resources/alma-fix/990050000600206441.json:271]: https://sigel.staatsbibliothek-berlin.de/suche?hit=3&q=DE-294-22%20&sort=sortby%20isl%2Fsort.ascending

Not sure why this is kicked out.

Comment (TobiasNx) [reply src/test/resources/alma-fix/990050000600206441.json:271]: Ah, the owner does not exist anymore. I could use the replacement isil instead.

**Review by dr0i — APPROVED** [review]

**Comment by dr0i** [comment]
> What are the chances that this will get merged before the next indexing?

It's deployed right now.

**Pull request merged**